packages/react/test/specs/components/Label/Label-test.tsx

```diff
@@ -13,14 +11,4 @@ describe('Label', () => {
   isConformant(Label)
   labelImplementsShorthandProp('icon', Icon, { mapsValueToProp: 'name' })
   labelImplementsShorthandProp('image', Image, { mapsValueToProp: 'src' })
-
-  describe('icon', () => {
```

Comment:
can you please provide a reason for removing this test as a comment?

Reply:
what do you mean? I changed the Label not to have tabindex 0 anymore. This test was failing as a result. What am I missing>

Reply:
got it, thx, fyi @mnajdova

Reply:
In this case, keyboard-only user will still be able to perform the action that the regular user can perform on click? I don't see how we are handling this.

Reply:
we need to do it some other way. having tabindex 0 just because the icon has onClick is not really the way to go.
1. we don't need tabindex 0 to have it clickable.
2. yes, the user can add tabindex 0 to the shorthand. 
3. it's not a button by default, so it should not be in the tab order by default.
4. we can think of some other way to do it.